src/core/server/models/settings/settings.ts

```diff
@@ -85,6 +87,71 @@ export type Auth = Omit<GQLAuth, "integrations"> & {
   integrations: AuthIntegrations;
 };
 
+export interface ExternalModerationPhase extends SigningSecretResource {
+  /**
+   * id identifies this particular External Moderation Phase.
+   */
+  id: string;
+
+  /**
+   * name is the name assigned to this ExternalModerationPhase for
+   * identification purposes.
+   */
+  name: string;
+
+  /**
+   * enabled when true, will use this phase in the moderation pipeline.
+   */
+  enabled: boolean;
+
+  /**
+   * url is the actual URL that should be called.
+   */
+  url: string;
+
+  /**
+   * format is the format of the comment body sent.
+   */
+  format: GQLCOMMENT_BODY_FORMAT;
+
+  /**
+   * timeout is the number of milliseconds that this moderation is maximum
+   * expected to take before it is skipped.
+   */
+  timeout: number;
+
+  /**
+   * createdAt is the date that this External Moderation Phase was created at.
+   */
+  createdAt: Date;
+}
+
+export interface ExternalExternalIntegration {
```

Comment:
this naming is kind of wild but I get where you're coming from. what if we changed this to just `ExternalIntegration` and changed `ExtrenalIntegration` to `ThirdPartyIntegration` to differentiate the two types of external? or maybe "customer integration" vs "third-party integration"?

Reply:
This feels like a renaming error, will take a look!